modules/33_CDR/portfolio/equations.gms

```diff
@@ -0,0 +1,168 @@
+*** |  AGPL-3.0, you are granted additional permissions described in the
+*** |  REMIND License Exception, version 1.0 (see LICENSE file).
+*** |  Contact: remind@pik-potsdam.de
+*** SOF ./modules/33_CDR/portfolio/equations.gms
+
+
+***---------------------------------------------------------------------------
+*** Equations concerning two or more options
+
+***---------------------------------------------------------------------------
+*'  CDR Final Energy Balance
+***---------------------------------------------------------------------------
+q33_demFeCDR(t,regi,entyFe)$(entyFe2Sector(entyFe,"cdr"))..
+	sum(fe2cdr(entyFe,entyFe2,te_used33),
+ 	 	v33_FEdemand(t,regi,entyFe,entyFe2,te_used33)
+	)
+  	=e=
+  	sum((entySe,te)$se2fe(entySe,entyFe,te),
+		vm_demFeSector_afterTax(t,regi,entySe,entyFe,"cdr","ETS")
+	)
+	;
+
+***---------------------------------------------------------------------------
+*'  Sum of all CDR emissions other than BECCS and afforestation, which are calculated in the core.
+***---------------------------------------------------------------------------
+q33_emiCDR(t,regi)..
+	vm_emiCdr(t,regi,"co2")
+	=e=
+	sum(te_used33, v33_emi(t,regi,te_used33))
+	;
+
+***---------------------------------------------------------------------------
+*'  Limit the amount of H2 from biomass to the demand without CDR.
+*'  It's a sustainability bound to prevent a large demand for biomass.
+***---------------------------------------------------------------------------
+q33_H2bio_lim(t,regi,te)$pe2se("pebiolc","seh2",te)..
+	vm_prodSE(t,regi,"pebiolc","seh2",te)
+	=l=
+    vm_prodFe(t,regi,"seh2","feh2s","tdh2s") - sum(fe2cdr("feh2s",entyFe2,te_used33), v33_FEdemand(t,regi,"feh2s",entyFe2,te_used33))
+	;
+
+***---------------------------------------------------------------------------
+*** DAC
+
+***---------------------------------------------------------------------------
+*'  Calculation of (negative) CO2 emissions from direct air capture.
+***---------------------------------------------------------------------------
+q33_DAC_capconst(t,regi)..
+	v33_emi(t,regi,"dac")
+	=e=
+	- sum(teNoTransform2rlf33("dac",rlf),
+		vm_capFac(t,regi,"dac") * vm_cap(t,regi,"dac",rlf)
+	)
+	;
+
+***---------------------------------------------------------------------------
+*'  Preparation of captured emissions to enter the CCS chain.
+*'  The first part of the equation describes emissions captured from the ambient air,
+*'  the second part calculates the CO2 captured from the gas used for heat production
+*'  assuming 90% capture rate.
+***---------------------------------------------------------------------------
+q33_DAC_ccsbal(t,regi,ccs2te(ccsCo2(enty),enty2,te))..
+	sum(teCCS2rlf(te,rlf), vm_ccs_cdr(t,regi,enty,enty2,te,rlf))
+	=e=
+	- v33_emi(t,regi,"dac")
+	+ (1 / pm_eta_conv(t,regi,"gash2c")) * fm_dataemiglob("pegas","seh2","gash2c","cco2") * sum(fe2cdr("fegas",entyFe2,te_used33), v33_FEdemand(t,regi,"fegas", entyFe2,te_used33))
+	;
```

Comment:
```
q33_ccsbal(t,regi,ccs2te(ccsCo2(enty),enty2,te))..
	sum(teCCS2rlf(te,rlf), vm_ccs_cdr(t,regi,enty,enty2,te,rlf))
	=e=
	-v33_emiDAC(t,regi)
	;
```

Reply:
As a note for potential further adjustment following from this:
In the remind2 reportEmi.R, emissions from CDR Fe-Demand are reported separately ("Emi|CO2|Energy|Demand|+|CDR (Mt CO2/yr)", section 2.1.3). For the respective industry value, indCCS is subtracted from the fe-demand emissions. For CDR, this is so far not done. I guess it is a very minor matter, but now that we have the distinction here, we can be precise and add it there

Reply:
I think it also has to be deducted in core/equations from "vm_emiTeMkt" (similar to vm_emiIndCCS) because it is part of the FEdemand emissions that are captured. 
(Previously, it was represented in vm_emiCdr, which now no longer is the case)

Reply:
Thanks for your comments! I noted all of them. However, for this already lengthy PR, I decided not to add this fix and report this bug in a separate issue so that we can better compare the previous CDR module with the refactored one.